_posts/2020-01-26-Pycon-de-writeup.markdown

```diff
@@ -0,0 +1,273 @@
+---
+layout: single
+title:  "Write-Up of the PyConDE & PyData Berlin 2019 conference"
+date:   2020-01-26 20:00:00 +0100
+tags: conference python talk
+header:
+  overlay_image: assets/images/background-banner-blur-conference-399158.jpg
+  overlay_filter: 0.2
+  show_overlay_excerpt: false
+author: Sebastian Neubauer
+author_profile: true
+---
+# Write-Up of the PyConDE & PyData Berlin 2019 conference
+
+
+The CfP for the 2020 PyCon.DE & PyData Berlin conference is now open (submit your proposal 
+[here](https://de.pycon.org/)). This is a great time to share our experiences from the 2019 conference and compile a 
+list of our highlight talks. Take these as inspirations for your own submission.
+
+
+PyCon.DE is a Python conference held in Germany where attendees can meet to learn about new and upcoming Python 
+libraries, tools, software, and data science.
+In 2019 PyCon.DE & PyData Berlin took place in Berlin and had more than 1000 participants basically doubling the amount 
+of attendees as compared to the 2018 conference in Karlsruhe. As almost all Python conferences this event was run and organised by 
+volunteers contributing free time and supported by community sponsors contributing work time of their employees. 
+Sebastian Neubauer from Blue Yonder is part of the organizing team.
+
+Python is basically used throughout the entire Blue Yonder stack and attending this conference allows to stay on top of 
+latest developments in the Python world. Furthermore, since Blue Yonder profits a lot from the open source community, 
+the company was a Gold sponsor at this year's conference and thus present with a booth at the venue. In total, eleven 
+Blue Yonder associates from the German offices in Karlsruhe and Hamburg traveled to the conference, talked to colleagues 
+and potential candidates at the booth, and learned new things attending the talks. Furthermore, three Blue Yonder 
```

Comment:
Second time in a row using _Furthermore_

Reply:
```suggestion
and potential candidates at the booth, and learned new things attending the talks. Additionally, three Blue Yonder 
```